lib/noticed.rb

```diff
@@ -16,6 +16,7 @@ module DeliveryMethods
     autoload :Database, "noticed/delivery_methods/database"
     autoload :Email, "noticed/delivery_methods/email"
     autoload :Slack, "noticed/delivery_methods/slack"
+    autoload :MicrosoftTeam, "noticed/delivery_methods/microsoft_team"
```

Comment:
This will need to be plural to match the delivery method key. Plus, Microsoft calls it "Teams", so we should make sure it matches. Sorry to be nitpicky! 😅

Reply:
I have tried doing that initially, but I ended up with an error with the class not being found. I'll have another crack in case it was something else causing it

Reply:
Looks like the `classify` in the run_delivery_method singularizes it. Might be able to just camel case it in some other way that doesn't lose the pluralization.

`"Noticed::DeliveryMethods::#{name.to_s.classify}".constantize`

Reply:
Looks like `camelize` instead of `classify` will do what we want. 👍 

Want to change that too?

Reply:
Shall do.

Reply:
Appreciate it!